app/styles/app.scss

```diff
@@ -159,12 +159,6 @@ tr.disabled {
   background-color: $porcelain;
 }
 
-.vl-modal-dialog {
-  display: block;
-  z-index: 40;
-  width: 80rem;
-}
-
 .vlc-scroll-wrapper__body.full-height {
   > .ember-view {
     height: 100% !important;
```

Comment:
Should check if it's really needed to use height 100% here. This seems a bit unnecessary.

Reply:
This code was not introduced by me, and subsequently I will not expand beyond the scope of the issue.